### DISCUSSION TITLE
How to target a local branch for testing in vagrant ?

### DISCUSSION THREAD

**Discussion by Cactusbone** [dokku/dokku#7154] [Q&A] [answered]
It seems to always pull branches from https://github.com/dokku, however I want to try a local branch ... 

I'm following [dokku/docs/getting-started/install/vagrant.md at master · dokku/dokku](https://github.com/dokku/dokku/blob/master/docs/getting-started/install/vagrant.md), to be able to use local testing

I'm having an issue with jq not being installed (issue soon to be opened) and wanted to try a fix before opening a PR, however the initial clone is done from https://github.com/dokku, which means I cannot start the vagrant container at all :( 

example with branch `fix-docker-arg` (from PR #7147)
```
    dokku-windows: Cloning into '/root/dokku'...
    dokku-windows: error: pathspec 'origin/fix-docker-arg' did not match any file(s) known to git
```

Any help greatly appreciated

**Comment by josegonzalez** [comment, marked as answer]
The windows version does a clone because of symlink issues. That was introduced in 2015 [here](https://github.com/dokku/dokku/pull/1490). If the normal vm works, you probably want to do that. Otherwise, what you are asking for is a limitation of VMs, not Dokku, and we can't really do anything to fix it.

**Reply by Cactusbone** [reply]
Noted ! I'll try the normal version then :)  
Thanks :)
